modules/ding_smartbanner/ding_smartbanner.info

```diff
@@ -0,0 +1,6 @@
+name = Ding Smartbanner
+description = Module detects mobile device and shows popup with link to native application.
+core = 7.x
+version = 7.x-4.1.0
```

Comment:
Normally we don't have the version in the info files as that would require it to be changes on new releases.

Reply:
This will be overriden when you release new version, with the script you use. I assume?

Reply:
No, modules don't have version. We only have versions in the profile in ding2.info